packages/react-ai/src/components/AIConversation/views/Controls/MessagesControl.tsx

```diff
@@ -73,20 +72,24 @@ export const MessageControl: MessageControl = ({ message }) => {
   return (
     <ContentContainer>
       {message.content.map((content, index) => {
-        return content.type === 'text' ? (
-          <TextContent data-testid={'text-content'} key={index}>
-            {content.value}
-          </TextContent>
-        ) : (
-          <MediaContent
-            data-testid={'image-content'}
-            key={index}
-            src={convertBufferToBase64(
-              content.value.bytes,
-              content.value.format
-            )}
-          ></MediaContent>
-        );
+        if (content.text) {
+          return (
+            <TextContent data-testid={'text-content'} key={index}>
+              {content.text}
+            </TextContent>
+          );
+        } else if (content.image) {
```

Comment:
There's no final else, I presume we don't want to render anything if it's not text or an image?

Reply:
That's correct at this time, there are other messages types like `toolUse` and `toolResult` that don't get rendered. I have another PR that will render custom components if they are found in the `toolUse` response